### PR TITLE
get rid of cgroupPaths for v2 case

### PR DESCRIPTION
**Pull request by kolyshkin**
This patchset fixes the remaining users of GetPaths() in v2 mode, and makes it panic for v2 controllers.

Similarly, make `GetUnifiedPath()` panic for v1 controllers since it must not be used in this case.

### CODE REVIEW

**Comment by kolyshkin** [comment]
There is one last user of GetPaths() for v2, it's `runc events`. This is addressed by #2352

**Comment by kolyshkin** [comment]
No longer a WIP, ready for review.

File: libcontainer/cgroups/cgroups.go [hunk 1]
```diff
@@ -40,9 +40,9 @@ type Manager interface {
 	// GetUnifiedPath returns the unified path when running in unified mode.
 	// The value corresponds to the all values of GetPaths() map.
 	//
-	// GetUnifiedPath returns error when running in hybrid mode as well as
+	// GetUnifiedPath panics when running in hybrid mode as well as
 	// in legacy mode.
-	GetUnifiedPath() (string, error)
+	GetUnifiedPath() string
```
Comment (AkihiroSuda) [inline libcontainer/cgroups/cgroups.go:45]: Lets split the interface

**Comment by kolyshkin** [comment]
Replaced by https://github.com/opencontainers/runc/pull/2386

**Pull request closed without merging**